code/dynamic_programming/src/Array_Median/median.cpp

```diff
@@ -0,0 +1,25 @@
+#include<iostream>
+#include<algorithm>
+using namespace std;
+
+float median(int n, int arr[n])
+{
+	if (n % 2 != 0)
+		return (float)arr[n / 2];
```

Comment:
Indentation should be made with 4 spaces, as specified in the style guide ([here](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B#indentations)).